inc/Cpu.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <cstdint>
+#include <map>
+#include <string>
+
+class Cpu {
+public:
+  Cpu() = default;
+  ~Cpu() = default;
+
+  void* GetRegister(std::string);
+  void RunInstruction(std::string);
+  void SetRegister(std::string, void*);
+
+private:
```

Comment:
Check the UML for new changes.

Reply:
I added the new Opcode member from the UML. Thanks